### PR TITLE
Added JUnit test and extracted Ui class

### PR DESCRIPTION
**Pull request by SharlynLui**
Extracted Ui class from GroceryList

### CODE REVIEW

File: src/main/java/exceptions/GitException.java [hunk 1]
```diff
@@ -7,7 +7,7 @@ public abstract class GitException extends Exception {
     protected String message;
 
     /**
-     * Constructs WallybotException.
+     * Constructs GiTException.
```
Comment (wallywallywally) [inline src/main/java/exceptions/GitException.java:10]: My bad, nice catch!

File: src/main/java/git/Ui.java [hunk 2]
```diff
@@ -63,6 +65,34 @@ public void displayHelp() {
         );
     }
 
+    public static void printExpSet(Grocery grocery) {
```
Comment (wallywallywally) [inline src/main/java/git/Ui.java:68]: I like the short and sweet methods!

File: src/main/java/grocery/GroceryList.java [hunk 3]
```diff
@@ -29,7 +30,7 @@ public void addGrocery(Grocery grocery) {
         }
         try {
             groceries.add(grocery);
-            System.out.println(grocery.getName() + " added!");
+            Ui.printGroceryAdded(grocery);
```
Comment (wallywallywally) [inline src/main/java/grocery/GroceryList.java:33]: I like the SLAP after integrating your methods!

File: src/test/java/grocery/GroceryListTest.java [hunk 4]
```diff
@@ -42,5 +44,17 @@ public void addGrocery_throwIllegalArgument_exception() {
         } catch (IllegalArgumentException e) {
             assertEquals("The grocery name is invalid.", e.getMessage());
         }
-    }    
+    }
+
```
Comment (wallywallywally) [inline src/test/java/grocery/GroceryListTest.java:48]: Short and sweet test!

File: src/test/java/grocery/GroceryListTest.java [hunk 5]
```diff
@@ -42,5 +44,17 @@ public void addGrocery_throwIllegalArgument_exception() {
...
+    public void removeGrocery_groceryDelete_exceptionThrown() {
+        try {
+            GroceryList gl = new GroceryList();
+            gl.addGrocery(new Grocery("fooood", null, null));
+            gl.removeGrocery("food");
+            fail("Expected NoSuchGroceryException not thrown");
+        } catch (NoSuchGroceryException e) {
+            assertEquals("The grocery does not exist!", e.getMessage());
+        }
+    }
```
Comment (64-1) [inline src/test/java/grocery/GroceryListTest.java:59]: Excellent test case!

**Review by 64-1 — APPROVED** [review]

**Review by wallywallywally — COMMENTED** [review]
LGTM, with many improvements to code readability!

**Review by wallywallywally — APPROVED** [review]

**Pull request merged**